tests/libtest.sh

```diff
@@ -71,6 +71,13 @@ wipe_signatures() {
 cleanup() {
   local vg_name=$1
   local devs=$2
+  local INFILE=/etc/sysconfig/docker-storage-setup
+  local OUTFILE=/etc/sysconfig/docker-storage
```

Comment:
INFILE and OUTFILE are local variables and should be small cap. That's the convention we are following.